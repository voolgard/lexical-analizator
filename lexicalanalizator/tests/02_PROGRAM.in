program dasha; var x: integer; begin x := 1 + 1; end.