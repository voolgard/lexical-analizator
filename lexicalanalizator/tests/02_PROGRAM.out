1	1	KEYWORD	program	program
1	9	IDENTIFIER	dasha	dasha
1	14	OPERATOR	;	;
1	16	KEYWORD	var	var
1	20	IDENTIFIER	x	x
1	21	OPERATOR	:	:
1	23	IDENTIFIER	integer	integer
1	30	OPERATOR	;	;
1	32	KEYWORD	begin	begin
1	38	IDENTIFIER	x	x
1	40	OPERATOR	:=	:=
1	43	INTEGER	1	1
1	45	OPERATOR	+	+
1	47	INTEGER	1	1
1	48	OPERATOR	;	;
1	50	KEYWORD	end	end
1	53	OPERATOR	.	.